### ISSUE TITLE
New update is broken

### ISSUE THREAD

**Issue by thenonamezz** [Chase-san/TiTsEd#47]
as i have written in the title the new update breaks the launcher.
whenever i try to boot the new update (0,19) it just stop and wont do anything.
then i install ver 0.18 again and it works fine

**Comment by algoRhythm99** [comment]
Going to need more than "it just stop and won't do anything."
Does it crash?
Do you get an error message?

**Comment by thenonamezz** [comment]
No nothing so crash so error message. It wont start or anything. It only loads for a few sec

**Comment by algoRhythm99** [comment]
So it exits cleanly without a crash message?
How do you know it starts at all, by looking at task manager?
What OS are you using?
What version of .NET Framework is installed?
How do you normally play TiTS, from a browser?
Can you link the save that is in the 1st slot?

**Comment by thenonamezz** [comment]
No crash message
It doesn't start at all
Windows 10 
4.6.1 i Think not sure
I download TiTS and play it from an external drive (never had any problems)
What 1st slot are you takling about?

**Comment by Chase-san** [comment]
Does this version work better? I rebuilt the current release on my system (where it is usually made).

[TiTsEd_v0.19.zip](https://github.com/Chase-san/TiTsEd/files/785686/TiTsEd_v0.19.zip)

**Comment by thenonamezz** [comment]
no it just loads for a bit which i can see by my mouse just like the normal one

**Comment by superdude808** [comment]
it might have to do with your anti virus thinking it's malware, avast seems to think it is atleast it's only done this to me this update had to disable avast so could load it up

**Comment by thenonamezz** [comment]
i use avast and i still cant start it even after i disabled it

**Comment by algoRhythm99** [comment]
This is probably Windows 10's SmartScreen getting in the way

**Comment by thenonamezz** [comment]
how? ver 0.18 works fine with no problems

**Comment by WelcomeToLevel2** [comment]
He could be right about the smartscreen thing.  I'm on win 8.1 and whenever i try to run it it pops up telling me that smartscreen stopped it from running

**Comment by Mister-Gregar** [comment]
I'm on windows 10, and it works fine.

**Comment by algoRhythm99** [comment]
I also use Windows 10, but I've seen that same behavior with other things.

**Comment by Knight176** [comment]
I've seen some people who have had the same problem with me. i use Norton anti-virus and when i went to load the program my anti-virus detected it as a Heur.advML.C. Virus and Completly Shut it down. i had to revert back to the v0.18 because of that. most displeasing

**Comment by Chase-san** [comment]
It might be due to a code change somewhere, since we tried compiling it on mine as well. We should review that we are not doing anything odd, or importing any packages which an antivirus might consider questionable.

**Comment by Chase-san** [comment]
We should remove any and all uses or references to

```
using System.Linq;
using System.Runtime.CompilerServices;
```

Mostly because I hate Linq and all it stands for, and we should not be using compiler services anywhere. One or both of these might be the cause of the anti-virus not being so chill anymore about TiTsEd.

**Comment by algoRhythm99** [comment]
Does this version work?
https://github.com/Chase-san/TiTsEd/releases/download/untagged-ac4bf640c72ff438f476/TiTsEd_v0.20.zip

**Comment by Knight176** [comment]
Clicked the Link and gave an Error 404 saying the page i was looking for was not found.

**Comment by algoRhythm99** [comment]
Try this:
https://github.com/Chase-san/TiTsEd/releases/download/v0.20/TiTsEd_v0.20.zip

**Comment by thenonamezz** [comment]
negative from here same as before

**Comment by Knight176** [comment]
able to DL it but when i tried it my anti- virus suddenly flagged it again from a high threat to a medium threat as a WS.Reputation.1 Virus. i suggest you look into the Stable v0.18 look at the code and compare it to your v0.20 and see the differences, it might show where something went wrong.

**Comment by algoRhythm99** [comment]
That isn't actually a virus, just means it has a low reputation score, meaning not that many people have used it.

**Comment by algoRhythm99** [comment]
I removed the updates check button/functionality.
https://github.com/Chase-san/TiTsEd/releases/download/v0.20/TiTsEd_v0.20.zip

**Comment by thenonamezz** [comment]
still nothing

**Comment by Chase-san** [comment]
Try compiling 0.18 on your end and see if that works for them.

**Comment by thenonamezz** [comment]
and how do i do that?

**Comment by Chase-san** [comment]
That was for algoRhythm.

**Comment by thenonamezz** [comment]
what do you mean with ''check out''

**Comment by algoRhythm99** [comment]
Try this one:
https://www.dropbox.com/s/r63mwm9k7ppo12y/TiTsEd_v0.18.zip?dl=0

This is the v0.18 built on my machine.

**Comment by thenonamezz** [comment]
negative it wont open

**Comment by algoRhythm99** [comment]
Right-click on the zip file, and select Properties,
Then check the "Unblock" box, and unzip it into a new location and try to run from there.

**Comment by thenonamezz** [comment]
it opened even the 0.20 ver. works

**Comment by algoRhythm99** [comment]
I think the only real solution for these issues is to get a certificate and sign the binary.

**Comment by thenonamezz** [comment]
so the only solution is ''unblocking'' it? you guys can do nothing about it?

**Comment by algoRhythm99** [comment]
Closing since there isn't much we can do about this other than getting the binaries signed and/or making an installer
